tock/hours/tests/test_views.py

```diff
@@ -4,6 +4,7 @@
 import json
 
 from django.core.urlresolvers import reverse
+from django.http import Http404
```

Comment:
Are you using this import somewhere and I'm not seeing it?

Reply:
Shoot, I'm not anymore - removing, thanks!